test/parallel/test-module-cache.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const fixture = require('../common/fixtures');
+
+const filePath = fixture.path('/test-module-cache.json');
```

Comment:
could we use `common/tmpdir` instead of a fixture, to avoid having an uncommitted file if the test fails?